src/CoreWCF.Primitives/src/CoreWCF/Description/OperationDescription.cs

```diff
@@ -124,9 +124,9 @@ public string Name
             get { return Behaviors; }
         }
 
-        internal KeyedByTypeCollection<IOperationBehavior> Behaviors { get; }
+        public KeyedByTypeCollection<IOperationBehavior> Behaviors { get; }
 
```

Comment:
Leave this internal and instead use the OperationBehaviors property which you can cast to `KeyedByTypeCollection<IOperationBehavior>` to access the Find method.

Reply:
This was addressed in 1b37a4a17c86f65c29e0dd8b14d01abe26179f33.